es6/index.js

```diff
@@ -0,0 +1,21 @@
+import got from 'got-promise';
+
+function pluckValue(response) {
+  return response.body[0].Value;
```

Comment:
Any reason we wouldn't want the whole object back? `response.body[0]`. Not sure really what the rest of that response looks like, but I assume Consul has a reason for exposing it to you when you get a key.